package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "@folio/stripes-testing": "^3.0.0",
-    "@folio/stripes-webpack": "^1.3.0",
+    "@folio/stripes-webpack": "^1.4.0",
```

Comment:
The build is unable to find stripes-webpack 1.4.0. I don't think that version exists yet..?

Reply:
That is correct, hence folio-org/stripes-webpack/pull/26.

Reply:
(Just added you as a reviewer there; sorry, forgot, REALLY gotta configure CODEOWNERS.)